terra-app-setup-chart/templates/network-policy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: deny-from-other-namespaces
+  namespace: {{ .Release.Namespace  }}
+spec:
+  podSelector:
+    matchLabels:
+  ingress:
+  - from:
+    - podSelector: {}
+    - namespaceSelector:
+        matchLabels:
+          name: nginx
```

Comment:
i don't think we currently label the `nginx` namespace (with label `name: nginx`) so we should probably add that before merging these changes

Reply:
I think we do label it now as part of one of @Qi77Qi's PRs.. (forget which one)

Reply:
oops didn't mean to mark as resolved - but sounds good

Reply:
So I don't think we do explicitly, but it seems to happen automatically. I confirmed by trying to add the label myself
<img width="711" alt="Screen Shot 2021-04-01 at 2 20 29 PM" src="https://user-images.githubusercontent.com/6465084/113337202-75724d00-92f5-11eb-82c2-6793ba427744.png">